src/main/java/de/tum/in/www1/artemis/domain/settings/ide/Ide.java

```diff
@@ -0,0 +1,47 @@
+package de.tum.in.www1.artemis.domain.settings.ide;
+
+import java.util.Collection;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.OneToMany;
+import jakarta.persistence.Table;
+
+import org.hibernate.annotations.Cache;
+import org.hibernate.annotations.CacheConcurrencyStrategy;
+
+import de.tum.in.www1.artemis.domain.DomainObject;
+
+@Entity
+@Table(name = "ides")
+@Cache(usage = CacheConcurrencyStrategy.NONSTRICT_READ_WRITE)
+public class Ide extends DomainObject {
+
+    @Column(name = "name", nullable = false)
+    private String name;
+
+    @Column(name = "deep_link", nullable = false, unique = true)
+    private String deepLink;
```

Comment:
What does this attribute store? The link that is used for cloning? Maybe add a comment for that